sbatch_singularity.cwl

```diff
@@ -127,7 +127,7 @@ requirements:
 
               # Format shell script
               shell_file = ['#!/bin/bash',
-                            '#SBATCH --partition=pascalnodes-medium',
+                            '#SBATCH --partition=pascalnodes',
                             '#SBATCH --job-name={submissionid}',
                             '#SBATCH --time=48:00:00',
```

Comment:
i think this will fail, i am pretty sure pascalnodes will only let you run for 12h, not 48.